examples/source_examples/sql_server/connector.py

```diff
@@ -0,0 +1,209 @@
+# This is a simple example for how to work with the fivetran_connector_sdk module.
```

Comment:
Please mention that the driver is to be installed locally for this to run to completion on debug